### PR TITLE
Add blog post on current state of storing sessions in Keycloak 26

### PR DESCRIPTION
**Pull request by mhajas**

### CODE REVIEW

File: blog/2024/storing-sessions-in-kc26.adoc [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+* Experimental and will be removed soon
+a|
+* When you used this setup with previous releases and cannot switch to persistent user sessions now
+a|
+----
+--features="cache-embedded-remote-store"
+--features-disabled="persistent-user-sessions"
+----
+.2+.^|Multiple sites (https://www.keycloak.org/high-availability/introduction[guide])
+|Persistent user sessions
```
Comment (pruivo) [inline blog/2024/storing-sessions-in-kc26.adoc:77]: consistent naming: On line 22, it says `Persistent sessions`. My suggestion is to use the same name everywhere (no preference between the two).

**Comment by pruivo** [comment]
@mhajas The table looks great to me :+1: Good work!

File: blog/2024/storing-sessions-in-kc26.adoc (hunk 1 above)
Comment (mhajas) [reply blog/2024/storing-sessions-in-kc26.adoc:77]: Thanks @pruivo, I changed all occurences of `persistent sessions` -> `persistent user sessions` to be on par with the name of the feature. Thanks!

**Review by ahus1 — APPROVED** [review]
Thank you for this blog post, LGTM. Let's publish it and and keep our fingers crossed that we get feedback on some of the discussions we linked here. 

Before publishing, please update the publishing date. Once it is published, I recommend posting this link to the SRE SIG group to get some first feedback.

**Pull request merged**